server/a_socials/settings/dev.py

```diff
@@ -26,4 +26,8 @@
     }
  }
 
+MEDIA_ROOT = os.path.join(BASE_DIR, 'static/media')
+MEDIA_URL = '/media/'
+MAX_UPLOAD_SIZE = 2097152
```

Comment:
Is this used anywhere?

Reply:
Was referencing the upload size from here before I added the `validate_image ` function. I'll remove that now.
But both MEDIA_ROOT and MEDIA_URL are in use as they specify the path to the directory that will hold user-uploaded files.